plugins/woocommerce/client/admin/client/wp-admin-scripts/email-editor-integration/index.ts

```diff
@@ -0,0 +1,14 @@
+// This file acts as a way of adding JS integration support for the email editor package
+
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+const NAME_SPACE = 'woocommerce/email-editor-integration';
+
+addFilter(
+	'mailpoet_email_editor_send_button_label',
+	NAME_SPACE,
+	() => 'Save WooCommerce email template'
```

Comment:
I see the string is not wrapped in the translation function, and when testing the editor I noticed it returned to the listing. I assume it is not a temporary button until we figure out how actually handle the block template email activation. Is this assumption correct?

Thinking about it now, I have an idea: we could perhaps use "Activate email" and "Deactivate email," which could set the email as active to replace the associated transactional email. But this is something we will solve in follow-up tickets.

Reply:
> I see the string is not wrapped in the translation function, and when testing the editor I noticed it returned to the listing. I assume it is ~not~ a temporary button until we figure out how actually handle the block template email activation. Is this assumption correct?

Yes, this assumption is correct.
I added a comment clarifying this in https://github.com/woocommerce/woocommerce/pull/55754/commits/c9a010081c8f3a28d53f4bbe61f996abcbfd784b#diff-189848a459676851e7c7e3afca00c5a6b45598df340444cdc2841ed31e9aef98L30